src/storage/exec/ScanNode.h

```diff
@@ -171,9 +171,15 @@ class ScanVertexPropNode : public QueryNode<Cursor> {
           break;
         }
       }
-      if (ret == nebula::cpp2::ErrorCode::SUCCEEDED &&
-          (filter_ == nullptr || QueryUtils::vTrue(filter_->eval(*expCtx_)))) {
-        resultDataSet_->rows.emplace_back(std::move(row));
+      if (ret == nebula::cpp2::ErrorCode::SUCCEEDED) {
+        if (filter_ == nullptr) {
+          resultDataSet_->rows.emplace_back(std::move(row));
+        } else {
+          auto result = QueryUtils::vTrue(filter_->eval(*expCtx_));
```

Comment:
The return type of` QueryUtils::vTrue` has changed

Reply:
Good catch, my bad …… I'll keep code in community version same as ent version